src/operations/drop.ts

```diff
@@ -86,21 +86,34 @@ export async function dropCollections(
       try {
         await executeOperation(db.client, dropOp, timeoutContext);
       } catch (err) {
+        // Note in FLE the error code is an Int32, where we expect a JS number, so we test
+        // the message as well here and below.
```

Comment:
I'm curious about this comment - could you elaborate a bit more on this?  All responses should be instances of `MongoDBResponse`, so the exact numeric type shouldn't matter.